### PR TITLE
Trigger benchmarks

### PR DESCRIPTION
**Pull request by chevdor**
Fix doc and triggers the benchmarking automation

### CODE REVIEW

**Review by gilescope — APPROVED** [review]

**Comment by gilescope** [comment]
bot merge

**Comment by chevdor** [comment]
ref. https://github.com/paritytech/ci_cd/issues/447

**Review by TriplEight — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -142,15 +142,4 @@ date to include them.
...
-3. Pull in the branch of Cumulus that has the version of Statemine you want to release
-4. Recompile `cargo build --profile production --locked --features runtime-benchmarks`
-5. From the root directory run `nohup ./scripts/benchmarks.sh &` (it will take quite a few hours)
-6. Checkout in your local machine to the branch of cumulus that has the version of Statemine you want to release
-7. `scp` from the host to your local machine the weights for Statemine, Westmint and Statemint you'll find in:
-   - `/parachains/runtimes/assets/statemine/src/weights`
-   - `/parachains/runtimes/assets/statemint/src/weights`
-   - `/parachains/runtimes/assets/westmint/src/weights`
-8. Commit the changes in your local and create a PR
+The benchmarks are trigger from PRs against the `release-parachains-v*` branches.
```
Comment (coderobe) [inline .github/ISSUE_TEMPLATE/release.md:145]:
```suggestion
The benchmarks are triggered by PRs against the `release-parachains-v*` branches.
```

**Review by coderobe — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/release.md (hunk 1 above)
Comment (coderobe) [reply .github/ISSUE_TEMPLATE/release.md:145]: why is this section in the issue template anyway

Comment (chevdor) [reply .github/ISSUE_TEMPLATE/release.md]: This is a NOOP (almost) to (re)trigger the benchmarks. You will find what you have in mind in [this PR](https://github.com/paritytech/cumulus/pull/1297)

**Comment by chevdor** [comment]
bot merge

**Comment by chevdor** [comment]
No longer required.

**Pull request closed without merging**